### DISCUSSION TITLE
How to deal with the lables after resizing the datasets in Project-MONAI / tutorials/detection?

### DISCUSSION THREAD

**Discussion by Dana136484** [Project-MONAI/tutorials#1603] [Q&A] [answered]
I need to train my spine datasets to obtain the boxes for each vertebra.The detection is not use the mothod of 'Resized', so the method of 'Resized' was used to process CT before training.But how to deal with the lables after resizing the datasets?Thanks!

**Comment by KumoLiu** [comment, marked as answer]
Hi @Dana136484, did you refer to the tutorial here?
https://github.com/Project-MONAI/tutorials/tree/main/detection

Hope it helps, thanks!

**Reply by Dana136484** [reply]
In the previous question, you have already helped me solve it.Thank you very much!

**Reply by Dana136484** [reply]
Hi,Kumo.Excuse me.  In tutorials detection.I am making the labels of bounding box. I would like to know the specific meaning of each of these 6 numbers.Thanks.
![1](https://github.com/Project-MONAI/tutorials/assets/126674286/b294dc4e-1472-40e3-a2c1-48ec1b4c4491)

**Reply by Dana136484** [reply]
Is it in the following format?
            "box": [
                [
                    xmin,
                    ymin,
                    zmin,
                    xmax-xmin,
                    ymax-ymin,
                    zmax-zmin
                ],
Thank you very much!

**Reply by KumoLiu** [reply]
https://github.com/Project-MONAI/tutorials/blob/30fbf22ec390f845a8953abe88ab68e06489a0cb/detection/config/config_train_luna16_16g.json#L2
https://github.com/Project-MONAI/MONAI/blob/8fa6931b14ba9617a595fff1d396ac44cc82e207/monai/utils/enums.py#L443

**Reply by Dana136484** [reply]
Thanks so much!

**Reply by Rajesh-ParaxialTech** [reply]
![image](https://github.com/Project-MONAI/tutorials/assets/159233894/c37aa906-9d3f-4b21-96df-e32075115e0a)
Hello!!

In the CCCWHD     ,   W  stands for width of the nodule,  H  for height and D  for depth.   And the units of dimension is mm ?  Please let me know.

Thank you

Rajesh
